src/Tribe/Tickets.php

```diff
@@ -555,10 +675,10 @@ public function has_permission( $post, $data, $nonce_action ) {
 		final public function ticket_add( $post_id, $data ) {
 			$ticket = new Tribe__Tickets__Ticket_Object();
 
-			$ticket->ID          = isset( $data['ticket_id'] ) ? absint( $data['ticket_id'] ) : null;
-			$ticket->name        = isset( $data['ticket_name'] ) ? esc_html( $data['ticket_name'] ) : null;
-			$ticket->description = isset( $data['ticket_description'] ) ? esc_html( $data['ticket_description'] ) : null;
-			$ticket->price       = ! empty( $data['ticket_price'] ) ? trim( $data['ticket_price'] ) : 0;
+			$ticket->ID             = isset( $data['ticket_id'] ) ? absint( $data['ticket_id'] ) : null;
+			$ticket->name           = isset( $data['ticket_name'] ) ? esc_html( $data['ticket_name'] ) : null;
+			$ticket->description    = isset( $data['ticket_description'] ) ? esc_html( $data['ticket_description'] ) : null;
+			$ticket->price          = ! empty( $data['ticket_price'] ) ? trim( $data['ticket_price'] ) : 0;
```

Comment:
We're sanitizing/escaping the other fields here, should we consider doing something beyond trimming this string?